src/main/java/org/opensearch/timeseries/MemoryTracker.java

```diff
@@ -360,4 +302,46 @@ public synchronized boolean syncMemoryState(Origin origin, long totalBytes, long
     public int getThresholdModelBytes() {
         return thresholdModelBytes;
     }
+
+    /**
+     * This function derives from the old code: https://tinyurl.com/2eaabja6
```

Comment:
This seems to be produced by the comparison tool -- but is exactly the same as before?

Reply:
it is not exactly the same, but adapted. Let me paraphrase.